yogstation/code/game/gamemodes/vampire/grave_fever.dm

```diff
@@ -4,8 +4,8 @@
 	stage_prob = 5
 	spread_text = "Non-Contagious"
 	spread_flags = DISEASE_SPREAD_SPECIAL
-	cure_text = "Antibiotics"
-	cures = list("spaceacillin")
+	cure_text = "Spaceacillin"
+	cures = list("/datum/reagent/medicine/spaceacillin")
```

Comment:
You might want to remove the ""'s or you're gonna have an even more embarassing follow-up PR